.bazelrc

```diff
@@ -31,6 +31,12 @@ build:devcontainer --disk_cache=/workspaces/magma/.bazel-cache
 common:devcontainer --repository_cache=/workspaces/magma/.bazel-cache-repo
 build:devcontainer --define=folly_so=1
 
+# MME FEATURES (default is agw_of)
+build --config=agw_of
+
+build:agw_of --per_file_copt=^lte/gateway/c/core/.*$@-DEMBEDDED_SGW=1,-DS6A_OVER_GRPC=1
+build:mme_oai --per_file_copt=^lte/gateway/c/core/.*$@-DEMBEDDED_SGW=0,-DS6A_OVER_GRPC=0
```

Comment:
The `--config=agw_of` seems to work for the build and the run looks normal.

But for `--config=mme_oai` we get the following error already during the `bazel build //lte/gateway/c/core:oai_mme --config=mme_oai` on vm and dev-container.

```
ERROR: /home/vagrant/magma/lte/gateway/c/core/BUILD.bazel:30:11: Compiling lte/gateway/c/core/oai/tasks/sgw/pgw_config.c failed: (Exit 1): gcc failed: error executing command /usr/bin/gcc -U_FORTIFY_SOURCE -fstack-protector -Wall -Wunused-but-set-parameter -Wno-free-nonheap-object -fno-omit-frame-pointer -g -MD -MF ... (remaining 405 argument(s) skipped)

Use --sandbox_debug to see verbose messages from the sandbox
lte/gateway/c/core/oai/tasks/sgw/pgw_config.c: In function 'pgw_config_process':
lte/gateway/c/core/oai/tasks/sgw/pgw_config.c:70:3: warning: implicit declaration of function 'async_system_command' [-Wimplicit-function-declaration]
   70 |   async_system_command(TASK_ASYNC_SYSTEM, PGW_ABORT_ON_ERROR,
      |   ^~~~~~~~~~~~~~~~~~~~
lte/gateway/c/core/oai/tasks/sgw/pgw_config.c:169:11: error: 'str_sgi' undeclared (first use in this function)
  169 |           str_sgi);
      |           ^~~~~~~
lte/gateway/c/core/oai/tasks/sgw/pgw_config.c:169:11: note: each undeclared identifier is reported only once for each function it appears in
Target //lte/gateway/c/core:oai_mme failed to build
Use --verbose_failures to see the command lines of failed build steps.
INFO: Elapsed time: 15.705s, Critical Path: 4.09s
INFO: 62 processes: 4 internal, 58 linux-sandbox.
FAILED: Build did NOT complete successfully
```

What is the `--config=mme_oai` intended to be used for? With `-DEMBEDDED_SGW=0` I would also expect to see the issue #12233 appear.

Reply:
Yes, this will need to be addressed :/ It is used for the OAI-Jenkins job, but I do not know other than that. @rdefosse / @lionelgo what is the use case for the flag?

Reply:
@LKreutzer Adding this as a blocker https://github.com/magma/magma/issues/12355

Reply:
Had a quick look:
* `pgw_config.c` with `EMBEDDED_SGW=0` is actually not working for the last 1.5y.
* its imo not failing with make because `EMBEDDED_SGW` also controls the set of build files - see `lte/gateway/c/core/oai/tasks/CMakeLists.txt`
* naive solution would be to have different targets :/
* at least this could be a pointer for libraries that can be refactored out of the mme binary target - @electronjoe would you agree?

Reply:
See https://github.com/magma/magma/pull/12365 for draft PR with more information